sklearn/naive_bayes.py

```diff
@@ -154,9 +164,17 @@ class GaussianNB(BaseNB):
     [1]
     """
 
-    def __init__(self, priors=None):
+    def __init__(self, classes=None, priors=None):
+        self.classes = classes
         self.priors = priors
 
+        # initialize additional parameter:
+        self.classes_ = None
```

Comment:
It's illegal to set trailing underscore attributes in ``__init__`` (and should result in some test failure)